indigo_app/static/javascript/indigo/views/annotations.js

```diff
@@ -541,28 +532,19 @@
         if (v.model === thread) {
           v.display();
 
-          this.gutter.activeItem = v;
-          // once it's rendered in the gutter, focus it
-          this.gutter.$nextTick(() => {
-            v.focus();
-            v.$el
-             .find('textarea')
-             .first()
-             .focus();
-          });
-        } else {
-          v.blur();
+          v.el.active = true;
+          window.setTimeout(() => {
```

Comment:
As discussed, let's add an event handler here for a new event from the gutter, something like 'layoutdone', and then set the textarea active once that is fired.